credentials/testdata/staging-unix.yaml

```diff
@@ -2,18 +2,13 @@ name: staging
 credentials:
   - name: read_file
     source:
-      path: testdata/someconfig.txt
+      path: $GOPATH/src/github.com/cnabio/cnab-go/credentials/testdata/someconfig.txt
```

Comment:
Was this change added to test the environment expansion ?

Reply:
Oops, let me revert that change it isn't necessary. It's an artifact of my previous efforts.